ui-v2/tests/unit/routes/dc-test.js

```diff
@@ -1,12 +1,12 @@
-import { moduleFor } from 'ember-qunit';
+import { module } from 'qunit';
+import { setupTest } from 'ember-qunit';
 import test from 'ember-sinon-qunit/test-support/test';
 
-moduleFor('route:dc', 'Unit | Route | dc', {
-  // Specify the other units that are required for this test.
-  needs: ['service:repository/dc', 'service:settings'],
-});
+module('Unit | Route | dc', function(hooks) {
+  setupTest(hooks);
 
-test('it exists', function(assert) {
-  let route = this.subject();
-  assert.ok(route);
+  test('it exists', function(assert) {
+    let route = this.owner.lookup('route:dc');
+    assert.ok(route);
+  });
 });
```

Comment:
Also these route tests… I pretty much never see them in recent Ember applications.